jwala-tomcat/src/main/resources/data/properties/vars.properties

```diff
@@ -56,4 +56,9 @@ jwala.binary.dir=../data/binaries
 jwala.agent.dir=../data/agent
 
 #jsch verbose logging flag
-ssh.verbose=false
\ No newline at end of file
+ssh.verbose=false
+
+#jwala client name

```

Comment:
To me, it appears that these parameters should be optional and we shouldn't provide default values here.
At the least, I would provide the keys with empty values and then have them commented out with a comment providing a description of where those values would get displayed when provided.

Reply:
done by @rahulsayini